tests/xtro-sharpie/common-AVFoundation.ignore

```diff
@@ -56,35 +56,13 @@
-!missing-null-allowed! 'AVFoundation.AVAssetTrack AVFoundation.AVPlayerItemTrack::get_AssetTrack()' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVAudioTime AVFoundation.AVAudioTime::ExtrapolateTimeFromAnchor(AVFoundation.AVAudioTime)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVMetadataItem[] AVFoundation.AVMetadataItem::FilterWithKey(AVFoundation.AVMetadataItem[],Foundation.NSObject,System.String)' is missing an [NullAllowed] on parameter #1
-!missing-null-allowed! 'AVFoundation.AVMetadataItem[] AVFoundation.AVMetadataItem::FilterWithKey(AVFoundation.AVMetadataItem[],Foundation.NSObject,System.String)' is missing an [NullAllowed] on parameter #2
-!missing-null-allowed! 'AVFoundation.AVMutableCompositionTrack AVFoundation.AVMutableComposition::AddMutableTrack(System.String,System.Int32)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'AVFoundation.AVMutableCompositionTrack AVFoundation.AVMutableComposition::CreateMutableTrack(AVFoundation.AVAssetTrack)' is missing an [NullAllowed] on return type
 !missing-null-allowed! 'System.Boolean AVFoundation.AVPlayerItem::Seek(Foundation.NSDate,AVFoundation.AVCompletion)' is missing an [NullAllowed] on parameter #1
-!missing-null-allowed! 'System.String AVFoundation.AVAssetTrack::get_ExtendedLanguageTag()' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'System.String AVFoundation.AVAssetTrack::get_LanguageCode()' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'System.String AVFoundation.AVAudioNode::GetNameForInputBus(System.nuint)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'System.String AVFoundation.AVAudioNode::GetNameForOutputBus(System.nuint)' is missing an [NullAllowed] on return type
-!missing-null-allowed! 'System.Void AVFoundation.AVAudioConverter::set_SampleRateConverterAlgorithm(System.String)' is missing an [NullAllowed] on parameter #0
-!missing-null-allowed! 'System.Void AVFoundation.AVPlayerItemMetadataOutput::SetDelegate(AVFoundation.IAVPlayerItemMetadataOutputPushDelegate,CoreFoundation.DispatchQueue)' is missing an [NullAllowed] on parameter #0
-!missing-null-allowed! 'System.Void AVFoundation.AVPlayerItemMetadataOutput::SetDelegate(AVFoundation.IAVPlayerItemMetadataOutputPushDelegate,CoreFoundation.DispatchQueue)' is missing an [NullAllowed] on parameter #1
-!missing-null-allowed! 'System.Void AVFoundation.AVPlayerItemMetadataOutputPushDelegate::DidOutputTimedMetadataGroups(AVFoundation.AVPlayerItemMetadataOutput,AVFoundation.AVTimedMetadataGroup[],AVFoundation.AVPlayerItemTrack)' is missing an [NullAllowed] on parameter #2
 !missing-null-allowed! 'System.Void AVFoundation.AVPlayerMediaSelectionCriteria::.ctor(AVFoundation.AVMediaCharacteristics[],AVFoundation.AVMediaCharacteristics[],System.String[])' is missing an [NullAllowed] on parameter #0
```

Comment:
Not sure why `[BindAs ...]` doesn't play nice with `[NullAllowed]`

```
		[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
		[Export ("initWithPrincipalMediaCharacteristics:preferredLanguages:preferredMediaCharacteristics:")]
		IntPtr Constructor ([NullAllowed] [BindAs (typeof (AVMediaCharacteristics []))]NSString[] principalMediaCharacteristics, [NullAllowed] [BindAs (typeof (AVMediaCharacteristics []))] NSString[] preferredLanguages, [NullAllowed] string[] preferredMediaCharacteristics);
```

Reply:
it's a bit trickier, the `[NullAllowed]` is primarily present to add (or not) a null check
the `[BindAs]` often change the type to something not nullable (by default)
but then you can get arrays...

there's some cases already in the bindings (try grep'ing for NullAllowed and BindAs)
or you can ignore it and we'll have a look later - it's still fine (to PR/merge) even if all xtro warnings are not removed :)

Reply:
I'll ignore it for now.